src/modules/jcms_rest/jcms_rest.services.yml

```diff
@@ -20,3 +20,6 @@ services:
     class: Drupal\jcms_rest\PathMediaTypeMapper
   jcms_rest.monitoring:
     class: eLife\Logging\Monitoring
+  plugin.manager.field.field_type:
+    class: Drupal\jcms_rest\FieldTypePluginManager
+    arguments: ['@container.namespaces', '@cache.discovery', '@module_handler', '@typed_data_manager']
```

Comment:
no idea what these arguments are, but I guess is the constructor that we are extending. Would be nice to leave here a link to the place where this list comes from

Reply:
These are the arguments required for the `FieldTypePluginManager`. This is just a way for me to override the Drupal core `core.services.yml` entry and intercept the `getPluginClass` method.

See: https://github.com/drupal/drupal/blob/8.3.7/core/core.services.yml#L603-L605